common/src/services/state.service.ts

```diff
@@ -1603,8 +1607,12 @@ export class StateService<TAccount extends Account = Account>
 
   async getLocale(options?: StorageOptions): Promise<string> {
     return (
-      await this.getGlobals(this.reconcileOptions(options, await this.defaultOnDiskLocalOptions()))
-    )?.locale;
+      (
+        await this.getGlobals(
+          this.reconcileOptions(options, await this.defaultOnDiskLocalOptions())
+        )
+      )?.locale ?? null
```

Comment:
Based on this https://github.com/bitwarden/jslib/blob/2341b1907ac601f70842e6a405cd641bf2706c06/common/src/models/domain/globalState.ts#L8 I'd expect it to default to `en`

Reply:
That's a good point, I'll look into locale and theme. This might just be because Desktop is behind on jslib at the moment and it may have been fixed already.

Reply:
I just checked and you are right, the global state does have defaults for those two now. The jslib version that desktop has now doesn't have that change, but Addison has a PR out now that brings that in
https://github.com/bitwarden/desktop/pull/1223

I'll remove the changes for the theme and locale, but we're still gonna need the change for clearClipboard

Reply:
@djsmith85 I removed those and this should be good to go now, thank you!